src/main/java/graphql/GraphQL.java

```diff
@@ -72,7 +70,7 @@ public ExecutionResult execute(String requestString, String operationName, Objec
             RecognitionException recognitionException = (RecognitionException) e.getCause();
             SourceLocation sourceLocation = new SourceLocation(recognitionException.getOffendingToken().getLine(), recognitionException.getOffendingToken().getCharPositionInLine());
             InvalidSyntaxError invalidSyntaxError = new InvalidSyntaxError(sourceLocation);
-            return new ExecutionResultImpl(Arrays.asList(invalidSyntaxError));
+            return new ExecutionResultImpl(Collections.singletonList(invalidSyntaxError));
```

Comment:
You are right for this specific line change i should've put into a separate pull request that fixes similar things. Will be more careful next time.